.github/workflows/tests-torch.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         pip install -r requirements/requirements-test.txt
         pip install torch==1.10.0+cpu --no-cache-dir -f https://download.pytorch.org/whl/cpu/torch_stable.html
         pip install -r requirements/requirements-pytorch.txt
+        pip install -r requirements/requirements-extras-cpflows.txt
```

Comment:
The requirements for tests have been revamped a bit. Maybe we can only have this for the specific model as a requirement?

Reply:
I will double check to make sure this is the optimal way to install test requirements after the recent changes.